### PR TITLE
VWE charge sniper tweaks

### PR DESCRIPTION
**Pull request by ItsSamGray**
## Changes

- Changed the VWE sniper rifle to fire 3 projectiles per shot

## Reasoning

- It is mentioned in the gun's description that it is a triple barreled sniper rifle like the "Triple Take" from Apex Legends so made the changes to make it so

## Alternatives

- Leave the gun single-shot

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [x] Playtested a colony (specify how long)

### CODE REVIEW

File: Patches/Vanilla Weapons Expanded/Ammo/ChargeTriplet.xml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+        <li Class="PatchOperationAdd">
+          <xpath>Defs</xpath>
+          <value>
+            <ThingDef ParentName="Base8x35mmChargedBullet">
+              <defName>Bullet_8x35mmCharged_Triple</defName>
+              <label>8x35mm Charged bullet</label>
+              <projectile Class="CombatExtended.ProjectilePropertiesCE">
+                <damageAmountBase>19</damageAmountBase>
+                <secondaryDamage>
```
Comment (N7Huntsman) [inline Patches/Vanilla Weapons Expanded/Ammo/ChargeTriplet.xml:36]: In theory, with the correct parent, most of this stuff should be inherited, meaning you don't have to duplicate it here.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
Can probably make the new ammoset more concise--I think a lot of this stuff can be inherited?

File: Patches/Vanilla Weapons Expanded/Ammo/ChargeTriplet.xml (hunk 1 above)
Comment (ItsSamGray) [reply Patches/Vanilla Weapons Expanded/Ammo/ChargeTriplet.xml:36]: The only difference is the pelletCount and the spreadMult which is inside the projectile class. Can the rest of the nodes inside the projectile class be inherited?

**Comment by Safairette** [comment]
> Can probably make the new ammoset more concise--I think a lot of this stuff can be inherited?

Inheriting requires the parent to have a Name attribute, which our non-base defs lack

**Review by N7Huntsman — APPROVED** [review]

**Pull request merged**